README.md

```diff
@@ -0,0 +1,20 @@
+Current functions:
```

Comment:
The README should be created using proper [Markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).

Additionally, the README should provide a clearer example of what the project does upfront, see the [Marketplace README](https://github.com/operator-framework/operator-marketplace/blob/master/README.md). It may also be helpful to point out that the library is in `ALPHA`.

Reply:
This was my README. We can remove it from this PR and create a proper one once we have more code in place.

Reply:
This is my README I threw together. Let's take it out of this branch and write it after we have a more defined API.